engine/src/test/java/org/hibernate/validator/test/internal/xml/Parent.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.test.internal.xml;
+
+/**
+ * Test class for HV-1534.
+ *
+ * @author robd
+ */
+public class Parent {
+
+	String parentAttribute = null;
```

Comment:
Please also add an attribute with a type argument constraint e.g. on String of `List<String>`. We need to have this working too.

Something like:
```
<getter name="parentListAttribute">
  <container-element-type type-argument-index="0">
    <constraint annotation="javax.validation.constraints.NotNull"/>
  </container-element-type>
</getter>
```
should do to declare the constraint.